### PR TITLE
[12.0][FWD] add generic exporter extracted from Magento connector

### PR DESCRIPTION
**Pull request by rousseldenis**
Forward-port of https://github.com/OCA/connector/commit/7d10c14cabcae9e0fe3b6e1151f87b1b49f2be10

### CODE REVIEW

**Comment by rousseldenis** [comment]
@lmignon @sebastienbeau

File: connector/components/synchronizer.py [hunk 1]
```diff
@@ -101,6 +110,310 @@ class Exporter(AbstractComponent):
...
 
 
+class GenericExporter(AbstractComponent):
+    """ Generic Synchronizer for exporting data from Odoo to a backend """
+    _name = 'generic.exporter'
+    _inherit = 'base.exporter'
+    _default_binding_field = None
+
+    def __init__(self, working_context):
+        super(GenericExporter, self).__init__(working_context)
```
Comment (sebalix) [inline connector/components/synchronizer.py:120]:
```suggestion
        super().__init__(working_context)
```

**Review by sebalix — APPROVED** [review]

**Review by guewen — APPROVED** [review]

File: connector/components/synchronizer.py (hunk 1 above)
Comment (rousseldenis) [reply connector/components/synchronizer.py:120]: Indeed. This is a simple forwardport to keep existing commits.

**Review by lmignon — APPROVED** [review]

**Comment by sebalix** [comment]
See my comment on https://github.com/OCA/connector/pull/338#issuecomment-498981158

**Pull request merged**